crates/nu-command/tests/commands/math/mod.rs

```diff
@@ -163,6 +163,77 @@ fn error_zero_division_decimal_decimal() {
     assert!(actual.err.contains("division by zero"));
 }
 
```

Comment:
It's better to add test cases for the following:
1.   `-3 div 2`
2.  `-3.0 div 2`